packages/react/src/molecules/pickers/radio/base.tsx

```diff
@@ -81,7 +82,11 @@ const PickerRadioBase = React.forwardRef((props: PickerRadioBaseProps, ref) => {
   const { options, addOrRemove, selected } = useSelect(
     props.onChange,
     props.dataSource,
-    props.defaultSelected
+    props.defaultSelected,
+    props.valueKey,
```

Comment:
@Subbu-betalectic pls pass here `props.defaultOpen` and `props.defaultQuery` from component props instead of `false` and `""`. No need to hard code values here as these variables are available in component's interface/props and default values are already  coded in `useSelect` custom hook..